### PR TITLE
Improve site accessibility score

### PR DESCRIPTION
**Pull request by SDBrett**
Fixes: #2094

Updates to site colours to align with VMware branding and improve contrast rations for site accessibility.

Update to youtube plugin for that it can will insert an iframes title as well. New usage of liquid template `{% youtube "<title>" %}

Updates to links to provide link text

Updates to images to add alt text.

Accessibility changes assist people visiting the site with visual impairments and improve the function of text to speech tools such as Jaws.

Signed-off-by: Brett Johnson <brett@sdbrett.com>

### CODE REVIEW

**Comment by SDBrett** [comment]
cc @jonasrosland

File: site/_includes/footer.html [hunk 1]
```diff
@@ -28,13 +28,13 @@ <h5>{{ site.footer.title }}</h5>
                 </ul>
             </div>
             <div class="col-12 col-md-4 text-center text-md-right">
-              <a href="/" alt="Homepage"><img src="/img/{{ site.logo }}" class="logo" /></a>
+              <a href="/" aria-label="Octant homepage"><img src="/img/{{ site.logo }}" class="logo" alt="Homepage"/></a>
```
Comment (jonasrosland) [inline site/_includes/footer.html]: This should say Velero homepage, not Octant :)

File: site/_scss/site/objects/_post.scss [hunk 2]
```diff
@@ -131,4 +131,16 @@
   img {
     max-width: 100%;
   }
-}
\ No newline at end of file
+}
+
+table {
```
Comment (jonasrosland) [inline site/_scss/site/objects/_post.scss]: This is only for blog posts, not documentation, correct? I don't think we've used tables in any blog posts yet, can you describe the change here?

File: site/resources.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ Here you will find external resources about Velero, such as videos, podcasts, an
 
 ## Previous community meetings
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/videoseries?list=PL7bmigfV0EqQRysvqvqOtRNk4L5S7uqwM" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+<iframe title="Velero Community Meeting - Oct 22, 2019" width="560" height="315" src="https://www.youtube.com/embed/videoseries?list=PL7bmigfV0EqQRysvqvqOtRNk4L5S7uqwM" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (jonasrosland) [inline site/resources.md]: That title should be "Velero Community Meetings/Open Discussions", as it's a playlist, not a single video.

File: site/_includes/footer.html (hunk 1 above)
Comment (SDBrett) [reply site/_includes/footer.html]: Ruined by copy / paste :p

File: site/_scss/site/objects/_post.scss (hunk 2 above)
Comment (SDBrett) [reply site/_scss/site/objects/_post.scss]: This was to fix tables not having boarders on the Contour envoy resource page, I included as part of this PR to keep layouts the same between OSS projects. 

I have noticed a side affect of this section of CSS on the Contour page, I'll resolve that and update this PR as well so it doesn't come into play. 

Comparison on the Contour page, left is without the snippet and right is the live site after the snippet was applied.

image

Comment (jonasrosland) [reply site/_scss/site/objects/_post.scss]: That looks a lot better!

**Comment by skriss** [comment]
Thanks for the PR @SDBrett! Only feedback I have here is that the links are now quite dark, and hard to distinguish from regular text -- see e.g. https://deploy-preview-2095--velero.netlify.com/community/ vs. https://velero.io/community/

**Comment by SDBrett** [comment]
I'll look into what I can do to make them stand out more.
Lightening will be a little limited due to contrasting, perhaps a different
font for links.

**Comment by SDBrett** [comment]
> Thanks for the PR @SDBrett! Only feedback I have here is that the links are now quite dark, and hard to distinguish from regular text -- see e.g. https://deploy-preview-2095--velero.netlify.com/community/ vs. https://velero.io/community/

I have updated to the PR to make it easier to distinguish from regular text. The new link colour still complies with VMware branding guidelines for colours and meets the 4.5:1 contrast ratio standard.

image

The update includes a change for the colour of links on a grey background section. Links within a `<div class="section-grey">` will use the colour defined as `$darkest-blue`. This additional change is due to insufficient contrast ratio on a grey background. 

image

**Pull request merged**